src/Symfony/Component/Validator/Tests/Fixtures/ConstraintChoiceWithPreset.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Symfony\Component\Validator\Tests\Fixtures;
+
+use Symfony\Component\Validator\Constraints\Choice;
+
+class ConstraintChoiceWithPreset extends Choice
+{
+    public $type;
+
+    public function __construct(
+        string $type
+    ) {
```

Comment:
```suggestion
    public function __construct(string $type) {
```

(as there is a single arg, but we don't do multiline function signatures at all in core anyway)